modules/localstack/v2/s3_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/aws/aws-sdk-go-v2/config"
 	"github.com/aws/aws-sdk-go-v2/credentials"
 	"github.com/aws/aws-sdk-go-v2/service/s3"
+	smithyendpoints "github.com/aws/smithy-go/endpoints"
```

Comment:
question: seems like unrelated changes?

Reply:
Sent in https://github.com/testcontainers/testcontainers-go/pull/2879

Once reviewed and merged, I'll rebase this one, thanks!